inst/assets/bibliography.bib

```diff
@@ -2276,6 +2276,61 @@ @article{Rahman2023
 	journal = {bioRxiv}
 }
 
+@article{qin2012_t2d,
+	title = {A metagenome-wide association study of gut microbiota in type 2 diabetes},
+	author = {Qin, Junjie and Li, Yingrui and Cai, Zhiming and Li, Shenghui and Zhu, Jianfeng and Zhang, Fan and Liang, Suisha and Zhang, Wenwei and Guan, Yuanlin and Shen, Dongqian and Peng, Yangqing and Zhang, Dongya and Jie, Zhuye and Wu, Wenxian and Qin, Youwen and Xue, Wenbin and Li, Junhua and Han, Lingchuan and Lu, Donghui and Wu, Peixian and Dai, Yali and Sun, Xiaojuan and Li, Zesong and Tang, Aifa and Zhong, Shilong and Li, Xiaoping and Chen, Weineng and Xu, Ran and Wang, Mingbang and Feng, Qiang and Gong, Meihua and Yu, Jing and Zhang, Yanyan and Zhang, Ming and Hansen, Torben and Sanchez, Gaston and Raes, Jeroen and Falony, Gwen and Okuda, Shujiro and Almeida, Mathieu and LeChatelier, Emmanuelle and Renault, Pierre and Pons, Nicolas and Batto, Jean-Michel and Zhang, Zhaoxi and Chen, Hua and Yang, Ruifu and Zheng, Weimou and Li, Songgang and Yang, Huanming and Wang, Jian and Ehrlich, S. Dusko and Nielsen, Rasmus and Pedersen, Oluf and Kristiansen, Karsten and Wang, Jun},
+	journal = {Nature},
+	volume = {490},
+	pages = {55--60},
+	year = {2012},
+	publisher = {Nature Publishing Group US New York},
+	doi = {10.1038/nature11450}
+}
+
+@article{mikropmlSoftware,
```

Comment:
we prefer using "lastnameyear" like "topguoclu2021" as the bibkey, would be good to update this here and elsewhere. It will also help to spot duplicate refs.